cmake/DftbPlusUtils.cmake

```diff
@@ -700,3 +683,27 @@ function (dftbp_create_lcov_targets lcov genhtml lcov_output_dir lcov_base_dir l
   endif()
 
 endfunction()
+
+
+# Checks whether the current compiler fullfills minimal version requirements.
+#
+#
+# Arguments:
+#   lang [in]: Language for which the compiler should be checked (e.g. Fortran, C, CXX)
+#   compiler_versions [in]: List with alternating compiler ids and minimal version numbers, e.g.
+#       "Intel;19.0;GNU;9.0". If the compiler is amoung the listed ones and its version number is
+#       less than the specified one, a fatal error message will be issued. Otherwise a simple status
+#       message is printed about the check.
+#
+function (dftbp_check_minimal_compiler_version lang compiler_versions)
+  while(compiler_versions)
+    list(POP_FRONT compiler_versions compiler version)
+    if("${CMAKE_${lang}_COMPILER_ID}" STREQUAL "${compiler}")
+      if (CMAKE_${lang}_COMPILER_VERSION VERSION_LESS "${version}")
+        message(FATAL_ERROR "${compiler} ${lang} compiler is too old (found \"${CMAKE_${lang}_COMPILER_VERSION}\" < \"${version}\")")
+      else()
+        message(STATUS "${compiler} ${lang} compiler is recent enough (found \"${CMAKE_${lang}_COMPILER_VERSION}\" >= \"${version}\")")
```

Comment:
Is this message relevant? CMake will print out the compiler version in any case.

Reply:
We can leave it away, indeed.